base/src/com/google/idea/blaze/base/dependencies/BlazeQuerySourceToTargetProvider.java

```diff
@@ -201,7 +210,27 @@ private static ImmutableList<TargetInfo> getTargetInfoList(
       return blazeQueryLabelKindParser.getTargets();
     } catch (IOException e) {
       throw new BlazeQuerySourceToTargetException("Failed to get target info list", e);
+    } finally {
+      if (!Registry.get("bazel.sync.keep.query.files").asBoolean()) {
```

Comment:
you can get boolean right from Registry without `asBoolean`

Reply:
How can I do it?

Reply:
Ok, done